vm.go

```diff
@@ -549,7 +549,7 @@ func init() {
 					if ret.Type() == LTNumber {
 						reg.SetNumber(RA, ret.(LNumber))
 					} else {
-						reg.SetNumber(RA, LNumber(0))
+						reg.Set(RA, ret)
```

Comment:
vim-go: Do not edit this file! /home/void/tmp/1111/gopher-lua/vm.go is a generated file.